### PR TITLE
Add v0.16.0 release post.

### LINKED ISSUE #51: Write blog post about the v0.16.0 release.

### PR DESCRIPTION
**Pull request by ni4**
Fixes #51

### CODE REVIEW

**Comment by ni4** [comment]
@ronaldtse @ribose-jeffreylau Does this style looks good? Instead of posting the whole changelog just write short summary (adding links to the examples page once it is available), and link to the whole changelog/release page at the bottom.

File: _posts/2022-01-20-rnp-release-0-16-0.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+The latest release of RNP v0.16.0 adds support for OpenSSL backend and security profiles, as well as improved security and compatibility with other implementations.
+
+== Other fixes and improvements:
+
+* Ability to disable certain features via compile-time switches (`ENABLE_AEAD`, `ENABLE_SM2`, etc.)
+* Mark signatures with SHA1/MD5 hash, produced after the specific date (2019-01-19 and 2012-01-01) as invalid
+* Fixed possible incompatibility with GnuPG on x25519 secret key export
+* Fixed export of non-FFI symbols export from the rnp.so/rnp.dylib
+* Fixed key expiration time calculation on some edge-cases
```
Comment (ribose-jeffreylau) [inline _posts/2022-01-20-rnp-release-0-16-0.adoc:22]:
```suggestion
* Fixed key expiration time calculation in some edge cases
```

File: _posts/2022-01-20-rnp-release-0-16-0.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+The latest release of RNP v0.16.0 adds support for OpenSSL backend and security profiles, as well as improved security and compatibility with other implementations.
+
+== Other fixes and improvements:
+
+* Ability to disable certain features via compile-time switches (`ENABLE_AEAD`, `ENABLE_SM2`, etc.)
+* Mark signatures with SHA1/MD5 hash, produced after the specific date (2019-01-19 and 2012-01-01) as invalid
+* Fixed possible incompatibility with GnuPG on x25519 secret key export
+* Fixed export of non-FFI symbols export from the rnp.so/rnp.dylib
```
Comment (ribose-jeffreylau) [inline _posts/2022-01-20-rnp-release-0-16-0.adoc:21]: Could the following work?

```suggestion
* Fixed export of non-FFI symbols from rnp.so/rnp.dylib
```

**Comment by ribose-jeffreylau** [comment]
> @ronaldtse @ribose-jeffreylau Does this style looks good? Instead of posting the whole changelog just write short summary (adding links to the examples page once it is available), and link to the whole changelog/release page at the bottom.

I think it is good.  Thanks @ni4 !

File: _posts/2022-01-20-rnp-release-0-16-0.adoc (hunk 1 above)
Comment (ni4) [reply _posts/2022-01-20-rnp-release-0-16-0.adoc]: Thanks, fixed!

File: _posts/2022-01-20-rnp-release-0-16-0.adoc (hunk 2 above)
Comment (ni4) [reply _posts/2022-01-20-rnp-release-0-16-0.adoc]: Thanks, fixed!

**Review by ribose-jeffreylau — APPROVED** [review]
👍

**Review by ronaldtse — APPROVED** [review]

**Comment by ronaldtse** [comment]
Thank you @ni4 !

**Pull request merged**